.github/CONTRIBUTING.md

```diff
@@ -4,87 +4,92 @@ Share your knowledge with the community by contributing to Magento DevDocs!
 You can contribute by creating an issue or pull request (PR) on our [DevDocs](https://github.com/magento/devdocs) GitHub repository.
 We welcome all types of contributions; from minor typo fixes to new topics.
 
-Magento's tech writer team and [Community Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/community-maintainers) review issues and pull requests on a regular basis, and we do our best to address all issues as soon as possible.
-Working through the backlog takes time, though, so we appreciate your patience.
+Magento's tech writer team and [Community Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/community-maintainers) review issues and pull requests on a regular basis. We do our best to address all issues as soon as possible, but working through the backlog takes time. We appreciate your patience.
 
 ## Rewards for contributions
 
 DevDocs works with Magento Community Engineering teams and projects.
-As you contribute PRs, you gain [Contribution Points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#points). We track [Contributors](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributors.html#/individual-contributors) and [Maintainers](https://devdocs.magento.com/guides/v2.3/contributor-guide/maintainers.html) with Community Engineering.
+As you contribute PRs, you gain [Contribution Points](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#points).
 
 If you write and contribute a full topic, we will add your name (or your company's name) at the top of the DevDocs page and link it to your blog or website!
 
 ## Get started
 
 ![Get started workflow](https://devdocs.magento.com/common/images/contribute-prerequisites.png)
 
-1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) to your account. Partners are required to add 2FA protection when contributing to Magento repositories.
+1. Make sure you have a [GitHub account](https://github.com/signup/free) with [Two-Factor Authentication](https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#two-factor)(2FA) enabled on your account. We require partners to add 2FA protection when contributing to Magento repositories.
```

Comment:
This is a note for a particular audience and can be skipped by general audience, so it should be represented as a separate note for partners. For example: _**Note for partners**: Add 2FA protection when contributing to Magento repositories._
Also, do we have a description of a contributor/maintainer/partner/etc roles somewhere? How do I know what role should I be in when reading the guidelines?

Reply:
Good question. This is for contributors only.

I wrote some maintainer guidelines a few weeks ago after onboarding a couple of new maintainers. I shared with them directly, but I've been meaning to make them public. I sent @lorikrell a copy a few days ago so she could add to the repo wiki, but she's been at Imagine all week. Instead of adding them to the wiki, maybe we should add to the repo as `.github/MAINTAINER-GUIDELINES.md`.

Reply:
See https://github.com/magento/devdocs/pull/4535